src/init/inst-functions.sh

```diff
@@ -1117,6 +1117,18 @@ TransferShared()
     find ${INSTALLDIR}/etc/shared -maxdepth 1 -type f -not -name ar.conf -not -name files.yml -exec mv -f {} ${INSTALLDIR}/etc/shared/default \;
 }
 
+checkDownloadContent()
+{
+    VD_FILENAME='vd_1.0.0_vd_4.8.0.tar.xz'
+
+    if [ "${DOWNLOAD_CONTENT}" = "yes" ]; then
```

Comment:
Shouldn't we add this variable in `install.sh` with a default value and document it to properly install by sources?

Reply:
Documentation was added to the end of etc/preloaded-vars.conf
Default value = n